staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -220,26 +220,18 @@ func Forbidden(field *Path, detail string) *Error {
 	return &Error{ErrorTypeForbidden, field.String(), "", detail}
 }
 
-// TooLong returns a *Error indicating "too long".  This is used to
-// report that the given value is too long.  This is similar to
-// Invalid, but the returned error will not include the too-long
-// value.
+// TooLong returns a *Error indicating "too long".  This is used to report that
+// the given value is too long.  This is similar to Invalid, but the returned
+// error will not include the too-long value. If maxLength is negative, it will
+// be included in the message.  The value argument is not used.
```

Comment:
If it's not used, can't we eliminate the argument?

I know, makes the PR larger, but this seems rather odd.

Reply:
Right, this is a staging repo. I suppose there are really projects which build a custom apiserver with its own hand-coded validation (haven't checked, though).

We could try to come up with new names, but that doesn't seem worth it.

/lgtm

Reply:
my thinking exactly